### PR TITLE
Parse Hashi-UI HTTPS environment variables to allow HTTPS for Docker

### LINKED ISSUE #270: HTTPS configuration for Hashi-UI does not support ENV variables
The ENV vars: HTTPS_ENABLE, SERVER_CERT and SERVER_KEY should be equivalent to the cli flags but they have no effect.

This should supposedly work, but from looking at the code, I think this was missed out. I'll submit a PR shortly

### PR DESCRIPTION
**Pull request by marcjay**
Fixes #270

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,9 +102,9 @@ hashi-ui can be controlled by both ENV or CLI flags as described below
 | `LOG_LEVEL` 	          | `log-level`               | `info`                  	| Log level to use while running the hashi-ui server - (`critical`, `error`, `warning`, `notice`, `info`, `debug`) |
 | `PROXY_ADDRESS`         | `proxy-address` 	      | `<empty>`               	| (optional) The base URL of the UI when running behind a reverse proxy (ie: example.com/nomad/)                   |
 | `LISTEN_ADDRESS`        | `listen-address`          | `0.0.0.0:3000`              | The IP + PORT to listen on |
-| `HTTPS_ENABLE`          | `https-enable`            | `false`                     | Use https protocol instead of http |
-| `SERVER_CERT`           | `server-cert`             | `<empty>`                   | Server certificate to use when https protocol is enabled |
-| `SERVER_KEY`            | `server-key`              | `<empty>`                   | Server key to use when https protocol is enabled |
+| `HTTPS_ENABLE`          | `https-enable`            | `false`                     | Use `--https-enable` or env `HTTPS_ENABLE=1` to enable HTTPS instead of HTTP for Hashi-UI |
```
Comment (lfarnell) [inline README.md]: I dont think the `HTTPS_ENABLE=1` to enable HTTPS is necessary. The first columns states this already

Comment (marcjay) [inline README.md]: To be honest, I added it to make it consistent with NOMAD_ENABLE and CONSUL_ENABLE. Appending '=1' when used as an env var wasn't obvious to me but happy to remove

Comment (lfarnell) [inline README.md]: Let's remove it and I will update the readme to make it clearer. How does that sound?

Comment (marcjay) [inline README.md]: Sure, done, and tidied up a loose |

**Comment by lfarnell** [comment]
Thanks for the changes!

**Comment by marcjay** [comment]
Thanks for the merge!

**Pull request merged**